packages/eslint-config-spruce/index.js

```diff
@@ -52,12 +57,6 @@ module.exports = {
 						}
 					}
 				],
-				'@typescript-eslint/explicit-function-return-type': [
```

Comment:
@kengoldfarb It was easier to just disable this... I figure individual projects can opt into `strict` TS, which would catch all of these issues.